stellar/contract.js

```diff
@@ -104,6 +116,30 @@ if (require.main === module) {
             mainProcessor(restoreInstance, contractName, [], options);
         });
 
+    program
+        .command('is_paused')
+        .description('Check if the contract is paused')
+        .argument('<contract-name>', 'contract name to check paused')
+        .action((contractName, options) => {
+            mainProcessor(handlePauseOperation, contractName, 'paused', options);
```

Comment:
```suggestion
            mainProcessor(isPausedCmd, contractName, 'paused', options);
```
isPausedCmd can call submitOperation internally while not causing much duplication